iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -148,34 +124,102 @@ class HomeViewController: UIViewController {
   private func handleTokenResponse(with result: Result<TokenDetails, TokenRequestError>) {
     switch result {
       case .failure(let error):
-        showAlert(with: error.localizedDescription)
+        showAlert(with: error.localizedDescription, title: "Error")
       case .success(let tokenDetails):
-        showAlert(with: tokenDetails.token)
+        showAlert(with: tokenDetails.token, title: "Success")
     }
   }
 
-  private func showAlert(with cardToken: String) {
+  private func showAlert(with message: String, title: String = "Payment") {
     DispatchQueue.main.async {
       let alert = UIAlertController(title: "Payment",
-                                    message: cardToken, preferredStyle: .alert)
+                                    message: message,
+                                    preferredStyle: .alert)
       let action = UIAlertAction(title: "OK", style: .default) { _ in
-        alert.dismiss(animated: true, completion: nil)
+        alert.dismiss(animated: true)
       }
       alert.addAction(action)
-      self.present(alert, animated: true, completion: nil)
+      self.present(alert, animated: true)
+    }
+  }
+
+  // MARK: - Apple Pay
+
+  private func handleApplePayAction() {
+    guard let paymentAuthorizationVC = ApplePayCreator.createPaymentAuthorizationViewController(delegate: self) else {
+      // Retrieve sample token from file and pass to `processToken`
+      guard let paymentData = getSampleDataResponseFromJSONFile() else { return }
+      processToken(paymentData)
+      return
+    }
+    present(paymentAuthorizationVC, animated: true)
+  }
+
+  private func processToken(_ paymentData: Data) {
```

Comment:
At some point we should also update ReadMe file & public docs to match our sample code. If there's any task for next release to cover any docs update for next release, could you include this? Otherwise, create one and ask @patrick-hoban-cko to tag for next release?

Reply:
On our ReadMe and docs we're only showing how to process payment data. Reason is we didn't show in our project how to get that token in the first place.

Since now we do more, we should showcase it on ReadMe.